airflow/hooks/jdbc_hook.py

```diff
@@ -57,3 +57,16 @@ def set_autocommit(self, conn, autocommit):
         :return:
         """
         conn.jconn.setAutoCommit(autocommit)
+
+    def get_autocommit(self, conn):
```

Comment:
Please add a test so that `getAutoCommit` has been called when calling `get_autocommit`. I know it might seems superflous but if the function gets refactored (or changed) in the future we need to make sure to not break sth. / breaking backwards compatiblity, etc. :)

Reply:
Would also be nice if you could do it for the set_autocommit as well 👍 (see [here](https://codecov.io/gh/apache/airflow/src/master/airflow/hooks/jdbc_hook.py#L59))